package/MDAnalysis/core/selection.py

```diff
@@ -247,6 +247,23 @@ def __init__(self):
         if self.periodic:
             self.apply = self._apply_distmat
 
+    def validate_dimensions(self, dimensions):
+        r"""Check if the system is periodic.
+
+        Parameters
+        ----------
+        dimensions : numpy.ndarray
+            6-item array denoting system size and angles
+
+        Returns
+        -------
+        None or numpy.ndarray
+            Returns argument dimensions if system is periodic, otherwise
```

Comment:
This description "if system is periodic" is ambiguous because `self.periodic` is an external choice if periodicity ought to be taken into account. Can you please say more clearly what goes into deciding if `None` or the box is returned? 

Also make clear that we only consider full 3D periodicity as periodic, we don't support periodicity in only some dimensions (or arbitrary space group operators... that would be fun).

Reply:
Then the wording can be changed "periodic in all three dimensions" and the validation code can be changed to:
```
if self.periodic and all(dimensions[:3]):
    return dimensions
return None
```
What do you think?

Reply:
Yes, sounds good.